sdk-setup/src/mb2

```diff
@@ -3587,6 +3599,14 @@ run_check() {
     return $rc
 }
 
+maybe_run_check() {
+    if [[ ! $BUILD_NORPMLINT ]]; then
```

Comment:
The use of `BUILD_*` variable to control the behavior implies that the function is only used from `run_build`, so given that it is just few lines long, I would suggest to implement it directly under `run_build` instead.

Reply:
Sure, if that's how you'd prefer it